guides/patterns/index.md

```diff
@@ -0,0 +1,8 @@
+# MarsBased Patterns
+
+A collection of MarsBased programming patterns in different languages.
+
+## Ruby on Rails
+
+- [Presenter](rails/presenter.md)
+- [Command](rails/command.md)
```

Comment:
Can we add the Pattern Query Object too, please 🙏 (I love It!)

https://github.com/MarsBased/patterns/commit/f771d64c5412d11c4d6a372c8168c89ed3391ba8

Reply:
I only need a re-review there before adding it in this repo. I moved the structure from multiple files to a single Markdown and I'd like new approvals.